e2e-tests/portal/cypress/e2e/journeys/read-only/fixtures/dealWithInvalidBondCoverStartDate.js

```diff
@@ -0,0 +1,17 @@
+const dealReadyToSubmitForReview = require('./dealReadyToSubmit');
+const dateConstants = require('../../../../../../e2e-fixtures/dateConstants');
+
+module.exports = () => {
+  const invalidCoverStartDate = `${dateConstants.yesterdayUnix}000`;
```

Comment:
I know there are occurrences of this coding style in the source code, could you please add a TODO to standardise this (should be covered in a separate PR).